CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 2.0.0-alpha.3
+- feat: Adds parameter arguments to unsafe database queries.
+- fix: Return objects are removed from database delete methods.
+- fix: Retrieves and removes future call entries in a single operation.
+- fix: Improves server only field validation.
+- fix: BREAKING. Removes automatic redirect from Relic.
+- fix: BREAKING. Removes `SerializationManager` as a parameter from `fromJson` factory constructor.
```

Comment:
Add the following:

- fix: BREAKING. Remove allToJson method.
- fix: toJson now includes all fields of Serverpod Models.